dotcom-rendering/index.d.ts

```diff
@@ -620,21 +635,13 @@ type FEContainerType =
 	| 'news/most-popular';
 
 type FEContainerPalette =
-	| `Branded`
 	| `EventPalette`
 	| `SombreAltPalette`
 	| `EventAltPalette`
 	| `InvestigationPalette`
 	| `LongRunningAltPalette`
 	| `LongRunningPalette`
 	| `SombrePalette`
-	| `Canonical`
-	| `Dynamo`
-	| `Special`
-	| `DynamoLike`
-	| `Special`
-	| `Breaking`
-	| `Podcast`
```

Comment:
After looking into this area more I see we don't support these other tags as 'palettes' so they don't need to exist here